engine/src/flutter/shell/platform/darwin/common/framework/Source/Logger.swift

```diff
@@ -0,0 +1,132 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Darwin
+import Foundation
+
+/// The level of logging severity.
+///
+/// These levels are used by `Logger` to determine if a message should be output.
+/// They are ordered by increasing severity.
+@objc(FlutterLogLevel) public enum LogLevel: Int {
+  /// Informational messages that are helpful for tracing application flow.
+  case info = 0
```

Comment:
nit: i believe `= 0` is the default, so can omit here.

Reply:
It's default, but because these are used in integer comparisons, it's potentially useful for readability to write them out explicitly, as we also do in [log_level.h](https://github.com/flutter/flutter/blob/master/engine/src/flutter/fml/log_level.h), even though enums in C/C++ (and most languages) are 0-based.

While the goal is to replace FML logging, realistically a bunch of people work across embedders in Flutter and it's likely useful to keep our log levels and their meanings in sync, and if we're going to do that, then may as well keep the values matching. 

Might be useful though to link back to fml/log_level.h in a comment though.